.github/workflows/lychee_url_checker.yml

```
@@ -10,9 +10,15 @@ on:
     # Run everyday at 3 am UTC
     - cron: "0 3 * * *"
 
+
+permissions: {}
+
 jobs:
   linkChecker:
     runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      issues: write
```

```suggestion
    permissions:
      contents: read
```

I don't think Lychee needs to write to an issue?